pkg/model/logstore/logstore.go

```diff
@@ -41,6 +41,15 @@ type LogSegment struct {
 
 	// Continues a line from a previous segment.
 	ContinuesLine bool
+
+	// When we store warnings in the LogStore, we break them up into lines and
+	// store them as a series of line segments. Atom marks the beginning of a
```

Comment:
What does Atom mean here?

Reply:
oooooops, Atom was an earlier name for this property before I changed it to 'anchor' (i changed it a few times)

Reply:
good catch, fixed!